scripts/validate_docstrings.py

```diff
@@ -512,6 +512,12 @@ def validate_one(func_name):
         examples_errs = doc.examples_errors
         if examples_errs:
             errs.append('Examples do not pass tests')
+        if 'import numpy as np' in ' '.join(doc.examples):
```

Comment:
I think it'd be good to use the module `doctest` to extract the content of the code in the examples. Meaning that something like this:
```
Examples
--------
This example does not import pandas as pd
>>> pd.Series(['pd is assumed to be imported as we want'])
```
should not generate an error.

And probably you want to include an example like this in the tests.